src/eval.ml

```diff
@@ -274,7 +290,11 @@ and tree_walk : dtree -> stack -> (term * stack) option = fun tree stk ->
                 match abstraction with
                 | None         -> default ()
                 | Some(fv, tr) ->
-                    let nfv = Bindlib.new_var mkfree (Bindlib.name_of fv) in
+                    let nfv = Bindlib.new_var mkfree Tree.var_prefix in
```

Comment:
Why using Tree.var_prefix here? Shouldn't Tree.var_prefix use in graphviz rendering only? If so, then Tree.var_prefix should be move to tree_graphviz.ml too.